package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "ml-testing-toolkit-ui",
-  "version": "15.5.3",
+  "version": "15.5.0",
   "description": "Mojaloop Testing Toolkit Web User Interface",
   "main": "index.js",
+  "proxy": "http://localhost:4040",
```

Comment:
Is this required?

Reply:
yes actually for hitting api endpoints live at pisp-ttk-sandbox docker container, either way it was giving cors errors